esp-hal-procmacros/Cargo.toml

```diff
@@ -35,8 +34,6 @@ syn               = { version = "2.0.100", features = ["extra-traits", "full"] }
 termcolor         = "1.4.1"
 
 [features]
-## Provide a `#[main]` procmacro to mark the entry point for Embassy applications.
-embassy = []
```

Comment:
Not sure what was the intent behind the embassy feature but I'm not sure it's worth keeping. It's not gating dependencies, so I think we can just provide the macro by default.